highlight.io/pages/docs/[[...doc]].tsx

```diff
@@ -454,7 +461,14 @@ export const getStaticProps: GetStaticProps<DocData> = async (context) => {
 			slug: currentDoc.simple_path,
 			relPath: currentDoc.rel_path,
 			docIndex: currentDocIndex,
-			docOptions: docPaths,
+			docOptions: docPaths.map((d) => {
```

Comment:
looks like there still is more to be pruned? ie. 

https://highlight-landing-git-docs-large-isr-payloads-highlight-run.vercel.app/_next/data/9gOeHPspx9B6ecXd4Ljzk/docs/getting-started/backend-sdk/dotnet.json?doc=getting-started&doc=backend-sdk&doc=dotnet


<img width="1632" alt="Screenshot 2024-08-16 at 17 25 56" src="https://github.com/user-attachments/assets/6bf1c5d9-4c41-4b6b-a0d2-ad31a163b676">

i think this is specific to the quickstartcontent including all of the other pages contents

<img width="1772" alt="Screenshot 2024-08-16 at 17 26 39" src="https://github.com/user-attachments/assets/247fce6a-b9e3-4611-a0d5-ac69e398c090">

Reply:
I see, let me take a look